spring-core/src/main/java/org/springframework/core/convert/support/GenericConversionService.java

```diff
@@ -166,6 +167,23 @@ public boolean canBypassConvert(@Nullable TypeDescriptor sourceType, TypeDescrip
 		return (converter == NO_OP_CONVERTER);
 	}
 
+	@SuppressWarnings("unchecked")
+	@Nullable
+	public <S, T> List<T> convertList(final @Nullable List<S> sourceList, final Class<T> targetClass) {
+
+		if (sourceList == null) {
+			return null;
+		}
+
+		if (sourceList.isEmpty()) {
+			return new ArrayList<>();
```

Comment:
Collections.emptyList() here, as it may return an already existing list

Reply:
Gonna change. That makes sense.